testgrid/cluster/entomologist_deployment.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: testgrid-entomologist
+  labels:
+    component: entomologist
+    app: testgrid
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: entomologist
+        app: testgrid
+    spec:
+      containers:
+        - name: entomologist
+          image: gcr.io/k8s-testgrid/entomologist:latest
+          imagePullPolicy: Always
+          args:
+            - --output=gs://k8s-testgrid-canary
+            - --gcs-credentials-file=/etc/gcp/k8s-testgrid.json
+            - --github-org=kubernetes
+            - --github-repo=test-infra
+            - --github-token-path=/etc/github/token.txt
+            - --github-endpoint=http://ghproxy
+          volumeMounts:
+            - name: gcp-account
+              mountPath: /etc/gcp
+              readOnly: true
+            - name: github-account
+              mountPath: /etc/github
+              readOnly: true
+      volumes:
+        - name: gcp-account
+          secret:
+            secretName: testgrid-service-account
+        - name: github-account
+          secret:
+            secretName: github-service-account
+---
+### Gh Proxy Deployment
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  namespace: default
+  labels:
+    app: ghproxy
+  name: ghproxy
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  # gce-ssd-retain is specified in prow/cluster/gce-ssd-retain_storageclass.yaml
+  #
+  # If you are setting up your own Prow instance you can do any of the following:
```

Comment:
Is this needed?

Reply:
Yes. If you're installing this on a brand-new cluster, knowing to apply that file does matter